### ISSUE TITLE
libraryVC title don't modify when change album

### ISSUE THREAD

**Issue by boboxiaodd** [Yummypets/YPImagePicker#40]
YPPickerVC.swift

```swift
    func navBarTapped() {
        
        let vc = YPAlbumVC()
        vc.noVideos = !self.configuration.showsVideoInLibrary
        let navVC = UINavigationController(rootViewController: vc)

        vc.didSelectAlbum = { [weak self] album in
            self?.libraryVC?.setAlbum(album)
            self?.libraryVC?.title = album.title   //add this code to fixed issue
            self?.libraryVC?.refreshMediaRequest()
            self?.setTitleViewWithTitle(aTitle: album.title)
            self?.dismiss(animated: true, completion: nil)
        }
        present(navVC, animated: true, completion: nil)
    }
```

**Comment by s4cha** [comment]
@boboxiaodd You mean the `title` property right ? 
Can you confirm the displayed title is indeed updated on your side ?


Cheers,

**Comment by boboxiaodd** [comment]
@s4cha 
first in library page, change Album, 
then nav to camera  , 
and back to library , the navbar 's title restore to old

in func updateUI

https://github.com/Yummypets/YPImagePicker/blob/80f49bc8579f7b761a17be037bed31f494e96e83/Source/YPPickerVC.swift#L269

**Comment by s4cha** [comment]
Ahhhhh!
My bad, I couldn't reproduce it at the time, I was missing the navigate and come back step :)
Your fix works as expected I am going to bump the version with it.
Nice one ;)

**Comment by s4cha** [comment]
@boboxiaodd that should be fixed in 2.7.1
https://github.com/Yummypets/YPImagePicker/releases/tag/2.7.1
Thanks for taking the time to report this and finding the fix :)
